docs/treefile.md

```diff
@@ -73,6 +73,11 @@ It supports the following parameters:
    An example use case for this is for Fedora CoreOS, which will blacklist the `python` and `python3`
    packages to ensure that nothing included in the OS starts depending on it in the future.
 
+ * `modules`: Object, optional: Describes RPM modules to install. Two keys are
+   supported:
+   * `install`: Array of strings, required: Set of RPM module specs to install
```

Comment:
I'm OK with this, but this is also related to e.g. https://github.com/coreos/rpm-ostree/issues/2584 and of course the larger https://github.com/coreos/rpm-ostree/issues/2326

If we had `packages-query` then it might be:

```
packages-query:
  module: cri-o:1.20
```
or so?  Then in theory we could also e.g. pin to specific repositories there too (not sure how often that would happen, but maybe).

Reply:
Or alternatively, maybe modules are on the balance more *not* packages then they are packages?  Dunno.

I'm fine with this as is too.

Reply:
Funny you mention https://github.com/coreos/rpm-ostree/issues/2584... I ended up also implementing this here (but will split it out) because it becomes a requirement for RHCOS to make use of the modular stuff.

Essentially, RHCOS composes cherry-pick non-modular packages that also appear in modules. And this usually is not possible: when you enable a module, every package in that module takes priority over the non-modular version of those packages. So for RHCOS, we need a way to tell rpm-ostree and libdnf that we want to override this and definitely pull e.g. non-modular `toolbox`. The new `repo-packages` key enable this.